### ISSUE TITLE
Custom Skeleton Application

### ISSUE THREAD

**Issue by nicholasmello** [xonmello/arcade-builder#35]
Custom skeleton is being applied as a separate directory rather than to the whole system

```
# ls /
bin              dev              home             lib64            lost+found       mnt              proc             run              sys              usr
custom-skeleton  etc              lib              linuxrc          media            opt              root             sbin             tmp              var
# ls /custom-skeleton/
etc
# ls /custom-skeleton/etc/
fstab    profile
```

**Comment by nicholasmello** [comment]
I was copying files to the wrong server... needs retest to make sure it is still an issue

**Comment by nicholasmello** [comment]
Fixed in  #63
